app/models/report.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Report < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true
+  validates :content, presence: true
+  has_many :comments, as: :commentable, dependent: :destroy
```

Comment:
```suggestion
  belongs_to :user
  has_many :comments, as: :commentable, dependent: :destroy
  
  validates :title, presence: true
  validates :content, presence: true
```

関連は関連でまとめて定義することが多いですね

Reply:
修正しました！